.gitignore

```diff
@@ -66,4 +66,4 @@ ssl/
 *.iws
 out/
 .idea_modules/
-
+Session.vim
```

Comment:
@ankitpal1029 please remove this

Reply:
Session.vim is autogenerated when using neovim much like .vscode